porting/nimble/src/os_mempool.c

```diff
@@ -336,7 +336,7 @@ os_mempool_info_get_next(struct os_mempool *mp, struct os_mempool_info *omi)
     omi->omi_num_blocks = cur->mp_num_blocks;
     omi->omi_num_free = cur->mp_num_free;
     omi->omi_min_free = cur->mp_min_free;
-    strncpy(omi->omi_name, cur->name, sizeof(omi->omi_name));
+    strlcpy(omi->omi_name, cur->name, sizeof(omi->omi_name));
```

Comment:
`strlcpy` is not a standard library function so keep `strncpy` and terminate manually instead

Reply:
@andrzej-kaczmarek Just pushed the changes using `strncpy` only after seeing travis build failures. Thanks :smiley: